<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherDialog</class>
 <widget class="QDialog" name="TeacherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="yspcclient.qrc">
    <normaloff>:/res/login/logo.png</normaloff>:/res/login/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#TeacherDialog
{
background-color: rgb(222, 222, 222);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>35</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>35</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="WidgetTop" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>77</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>77</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>35</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>77</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>77</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/res/teacher/pcindex_head.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>课程表</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>845</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="PButtonLogout">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
border:none;
font-size:15px;
}
QPushButton:hover
{
	background-color: rgb(200, 200, 200);
}</string>
        </property>
        <property name="text">
         <string>退出登录</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="WidgetLeftBottom" native="true">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
background-color: rgb(255, 255, 255);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="PictureWidget" name="PWidgetHead" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>240</height>
         </size>
        </property>
        <widget class="QWidget" name="WidgetNameConcem" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>170</y>
           <width>240</width>
           <height>70</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#WidgetNameConcem
{
background-color: rgba(28, 27, 27, 224);
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="5">
           <widget class="QLabel" name="LabelTeacherName">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel
{
	background: none;
	color: rgb(255, 255, 255);
	font-size:14px;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel
{
	background: none;
	color:  rgb(154, 154, 154);
	font-size:15px;
}
</string>
            </property>
            <property name="text">
             <string>已有</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="LabelConcemCount">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel
{
	background: none;
	color:  rgb(244, 179, 61);
	font-size:15px;
}
</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel
{
	background: none;
	color:  rgb(154, 154, 154);
	font-size:15px;
}
</string>
            </property>
            <property name="text">
             <string>人关注</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>79</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑 Light</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(46, 46, 46);</string>
          </property>
          <property name="text">
           <string>老师介绍</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="TEditIntroduce">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit
{
border:0px;
background-color:transparent;
color: rgb(138, 141, 150);
}
</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>40</number>
          </property>
          <item>
           <widget class="QRadioButton" name="RButtonCreateCourse">
            <property name="minimumSize">
             <size>
              <width>95</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>95</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton
{
	font: 16px &quot;微软雅黑&quot;;
	spacing: 10px;
}
QRadioButton::indicator 
{ 
    width: 20px;
    height: 20px;
	image: url(:/res/teacher/创建课程.png);
}
QRadioButton:hover
{
	background-color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string>创建课程</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RButtonUploadCourseware">
            <property name="minimumSize">
             <size>
              <width>95</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>95</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton
{
	font: 16px &quot;微软雅黑&quot;;
	spacing: 10px;
}
QRadioButton::indicator 
{ 
    width: 20px;
    height: 20px;
	image: url(:/res/teacher/上传课件.png);
}
QRadioButton:hover
{
	background-color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string>上传课件</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RButtonCourseSignUpRecord">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton
{
	font: 16px &quot;微软雅黑&quot;;
	spacing: 10px;
}
QRadioButton::indicator 
{ 
    width: 20px;
    height: 20px;
	image: url(:/res/teacher/课程报名记录.png);
}
QRadioButton:hover
{
	background-color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string>课程报名记录</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	background-color: rgb(222, 222, 222)
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>30</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="GregorianCalendarWidget" name="GCWidgetDate" native="true"/>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="WidgetDateCourse" native="true">
       <property name="minimumSize">
        <size>
         <width>530</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>530</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget
{
background-color: rgb(255, 255, 255);
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="LabelDate">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:16px;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="SAreaDateCourse">
          <property name="styleSheet">
           <string notr="true">QScrollArea
{
	border:none;
	border-top:1px solid rgb(200,200,200);
}
QScrollBar::vertical
{
	width: 10px;
	background:rgb(255,255,255);
	margin:0px 0px 0px 0px;
}
QScrollBar::handle:vertical
{
	background: rgb(200, 200, 200);
}
</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="SAWContentsDateCourse">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>470</width>
             <height>229</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>470</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>470</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="VLayoutDateCourse">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑 Light</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>我的课程</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="CBoxCourseType">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox
{
	border:transparent;
	background: transparent;
	padding: 1px 1px 1px 1px;
	font: 15px &quot;微软雅黑&quot;;
}
QComboBox:focus
{
	border:transparent;
	background: transparent;
	padding: 1px 1px 1px 1px;
}
QComboBox::drop-down
{
	width: 25px;
	background:transparent;
}
QComboBox::down-arrow 
{
	image: url(:/res/teacher/首页_下拉.png);
}
QComboBox QAbstractItemView
{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	selection-color: rgb(0, 0, 0);
	selection-background-color:rgb(85, 170, 255);
}
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QWidget" name="WidgetCourse" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget
{
background-color: rgb(255, 255, 255);
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QScrollArea" name="SAreaMyCourse">
          <property name="styleSheet">
           <string notr="true">QScrollArea
{
	border:none;
}
QScrollBar::vertical
{
	width: 10px;
	background:rgb(255,255,255);
	margin:0px 0px 0px 0px;
}
QScrollBar::handle:vertical
{
	background: rgb(200, 200, 200);
}
</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="SAWContentsMyCourse">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1038</width>
             <height>369</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>1038</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1038</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="VLayoutMyCourse">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="PagingWidget" name="PWidgetCoursePage" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureWidget</class>
   <extends>QWidget</extends>
   <header>picturewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PagingWidget</class>
   <extends>QWidget</extends>
   <header>pagingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GregorianCalendarWidget</class>
   <extends>QWidget</extends>
   <header>gregoriancalendarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="yspcclient.qrc"/>
 </resources>
 <connections/>
</ui>
